evalml/tests/component_tests/test_column_selector_transformers.py

```diff
@@ -168,3 +235,33 @@ def test_column_transformer_int_col_names_np_array(class_to_test, answers):
 
     transformer = class_to_test(columns=[])
     assert_frame_equal(answer3, transformer.transform(X))
+
+
+def test_dtype_column_transformer_ww_types():
```

Comment:
Can you add a test that selects based on capitalized types? ie `"NaturalLanguage"`, `"Double"`